src/Runner/CliTester.php

```diff
@@ -101,21 +107,23 @@ private function loadOptions()
     tester.php [options] [<test file> | <directory>]...
 
 Options:
-    -p <path>                    Specify PHP interpreter to run (default: php).
-    -c <path>                    Look for php.ini file (or look in directory) <path>.
-    -l | --log <path>            Write log to file <path>.
-    -d <key=value>...            Define INI entry 'key' with value 'val'.
-    -s                           Show information about skipped tests.
-    --stop-on-fail               Stop execution upon the first failure.
-    -j <num>                     Run <num> jobs in parallel (default: 8).
-    -o <console|tap|junit|none>  Specify output format.
-    -w | --watch <path>          Watch directory.
-    -i | --info                  Show tests environment info and exit.
-    --setup <path>               Script for runner setup.
-    --colors [1|0]               Enable or disable colors.
-    --coverage <path>            Generate code coverage report to file.
-    --coverage-src <path>        Path to source code.
-    -h | --help                  This help.
+    -p <path>                Specify PHP interpreter to run (default: php).
+    -c <path>                Look for php.ini file (or look in directory) <path>.
+    -l | --log <path>        Write log to file <path>.
+    -d <key=value>...        Define INI entry 'key' with value 'val'.
+    -s                       Show information about skipped tests.
+    --stop-on-fail           Stop execution upon the first failure.
+    -j <num>                 Run <num> jobs in parallel (default: 8).
+    -o <console|tap|junit|log|none>
+                             Specify one or more output formats with optional file name.
+                             (e.g. -o junit:output.xml)
+    -w | --watch <path>      Watch directory.
```

Comment:
What about `-o <console|tap|junit|log|none>  (e.g. -o junit:output.xml)` to save one row?